crane/crane.go

```diff
@@ -92,6 +94,19 @@ func checkDockerClient() {
 			printErrorf("Unsupported client version! Please upgrade to Docker %v or later.\n", intJoin(requiredDockerVersion, "."))
 		}
 	}
+	actualDockerVersion = versions
+}
+
+func validateDockerClientAbove(expectedVersions []int) bool {
```

Comment:
Apologies, I had forgotten [my own comment](https://github.com/michaelsauter/crane/pull/260#discussion_r53839650) :( It does introduce some complexity though... any opinion @michaelsauter ?